app/macos/Podfile.lock

```diff
@@ -149,4 +149,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: ce7dbe26c78bfc7ba46736094c1e2d25982870fa
 
-COCOAPODS: 1.12.1
+COCOAPODS: 1.12.1
```

Comment:
we really like messing with this file ...

Reply:
I am getting frustrated with this, the thing is it gets updated when you run it on your local for macos/ios platform. Can we add it in ```.gitignore``` list.